### PR TITLE
Add/about page header

### PR DESCRIPTION
**Pull request by santiagoejm**
The `AboutUsHeader` component is been defined at the `src/components` folder & been used in the new `src/pages/AboutUs` page.

Note: the route to the **About Us Page** it's not yet been defined in the `docusaurus.config.js` file, only manual access to this page is possible at the moment though the url:  _/aboutus_ 

The result of the header is: 

Screenshot 2022-12-15 at 17 34 42

### CODE REVIEW

File: src/components/AboutUsHeader/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  .information_container {
+    height: 20vw;
+    justify-content: space-evenly;
+    align-items: center;
+    text-align: center;
+  }
+}
+
+
+
```
Comment (Siegfriedk) [inline src/components/AboutUsHeader/styles.module.css:94]: Please create a new PR and remove too many new lines

**Review by Siegfriedk — CHANGES_REQUESTED** [review]
The background image is WAAAY to big. 

its over a megabyte and a 5 second online compression here https://imagecompressor.com/ already reduces it to 500kb

File: src/components/AboutUsHeader/styles.module.css (hunk 1 above)
Comment (santiagoejm) [reply src/components/AboutUsHeader/styles.module.css:94]: @Siegfriedk noted! creating new one right away

Comment (santiagoejm) [reply src/components/AboutUsHeader/styles.module.css]: Due to conflicts @Siegfriedk I see now why a complete new PR is requested :/

**Pull request closed without merging**